test/Helper.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity =0.8.17;
 
 import "forge-std/Test.sol";
-import "@uniswap/v3-core/contracts/Interfaces/IUniswapV3Pool.sol";
+import "@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol";
```

Comment:
lol such an unexpected bug. essentially the mac's filesystem is case insensitive unlike Linux'. Since github actions on linux containers, it failed.